src/App.jsx

```diff
@@ -1,9 +1,100 @@
+import { useState, useRef } from "react";
+
+import SideBar from "./components/SideBar";
+import Entry from "./components/Entry";
+import CreateProject from "./components/CreateProject";
+import Project from "./components/Project";
+
+const TEMP = [
+  {
+    title: "Vacation",
+    date: "2024-05-09",
+    description: "Tracking the tasks I’ll be doing on vacation.",
+    tasks: ['Learn React','Visit Grodno','Introduce girlfriend to parents']
+  }
+];
+
 function App() {
+  const [projects, setProjects] = useState(TEMP);
+  const [selectedProject, setSelectedProject] = useState(TEMP[0]);
+
+  const [isAddPrj, setIsAddPrj] = useState(false);  
+  const taskProject = useRef(null);
+
+  function handleAddPrj() {
+    setIsAddPrj(!isAddPrj);
+    setSelectedProject(undefined);
+  }
+
+  function handleCreateProject(objPrj) {
+    setProjects((prevProjects) => {
+      objPrj.tasks = [];
+      const updatedProjects =
+        [...prevProjects,
+          objPrj
+        ];
+
+      return updatedProjects;
+    });
```

Comment:
You can do like this:
objPrj.tasks = [];
return [...prevProjects, objPrj];

Or like this: 
return [...prevProjects, {...objPrj, tasks: []}];